app/frontend/src/locales/es/translation.json

```diff
@@ -6,6 +6,19 @@
     "login": "Iniciar sesión",
     "logout": "Cerrar sesión",
     "clearChat": "Borrar chat",
+    "history": {
+        "chatHistory": "Historial de chat",
```

Comment:
Did you use AI for the translations? i.e. do we need a human to review them?

Reply:
Okay, will do!

Reply:
I've asked a colleague to review the es translations. Wassim has reviewed fr.

Reply:
I approve of the Spanish translation 👍

Reply:
Thanks miguel! Resolving this thread.